diagnostic.md

```diff
@@ -5,74 +5,80 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+to handle and hold the information so that it can relay it
+to the front end to help interact with the client request.  example would be
+client wants to read a data the backend would gather and send it to front end to
+view
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+router
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+the controller
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+Views are merely puppets reading what the controller gives them.
 ```
 
 What does C.R.U.D stand for?
 
 ```md
-// your response here
+create read update destory
 ```
 
 In which part of the MVC pattern can we find C.R.U.D actions?
 
 ```md
-// your response here
+models
 ```
 
 List at least 5 standard rails actions that C.R.U.D requests correspond to?
 
 ```md
-// your response here
+GET, POST, PATCH, DELETE
```

Comment:
These are the HTTP verbs, we wanted the rails actions that they correspond to:

`Index, Show, Create, Update, and Destroy`